drivers/volume/csi/csi.go

```diff
@@ -634,14 +683,11 @@ func (c *csi) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.A
 		if contentName != "" {
 			vscError = c.getSnapshotContentError(contentName)
 		}
-
 		switch {
 		case volumeSnapshotReady && volumeSnapshotContentReady:
-			vInfo.Status = storkapi.ApplicationBackupStatusInCleanup
-			vInfo.Reason = "Backup uploaded, cleaning up snapshot objects"
+			vInfo.Reason = "Snapshot finished, pending completion of other snapshots"
```

Comment:
As discussed offline we'll revisit this in a future release to all individual volumes to be marked as Successful